joomla/joomla-cms
# [4.0] Allow scss compilation for any template

Pull Request for Issue # .

### Summary of Changes
Allow the scss compiler to transpile all the scss files [given that they exist in a folder named `scss`] in any template (frontend or administration). This allows the usage of the tools for developing custom templates.


### Testing Instructions
Run `npm run build:css` and check that both Cassiopeia and Atum are not visually broken. Also rename the configuration.php and check that the installation is also not visually broken


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required

No.
